CRT.c

```diff
@@ -706,3 +716,67 @@ void CRT_setColors(int colorScheme) {
 
    CRT_colors = CRT_colorSchemes[colorScheme];
 }
+
+void CRT_handleSIGSEGV(int signal) {
+   CRT_done();
+
+   fprintf(stderr, "\n\n"
+      "FATAL PROGRAM ERROR DETECTED\n"
+      "============================\n"
+      "Please report this issue at https://htop.dev including the following information:\n"
```

Comment:
Maybe use https://github.com/htop-dev/htop/issues/new as issue link?

Reply:
Or make https://htop.dev/new-issue point there. That's more flexible and allows for us to change where reports drop; even for ancient™ versions.